<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetWord</class>
 <widget class="QDialog" name="SetWord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set word</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTextEdit" name="oldTextEdit">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <italic>true</italic>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="newTextEdit">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <italic>true</italic>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>248</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="mode_comboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="selectButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setButton">
         <property name="text">
          <string/>
         </property>
         <property name="shortcut">
          <string>Ctrl+Return</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
